source/tutorial.txt

```diff
@@ -55,33 +42,18 @@ Next, complete any of the following clients to work with your new backend:
 - :doc:`A Node.js Command Line Interface (CLI) </tutorial/nodejs-cli>`
 
 .. _template-applications:
+.. _tutorial-task-tracker-create-realm-app:
+.. TODO remove the ref above once the other tutorials have been updated.
 
-Template Applications
----------------------
-
-Realm template applications are working example apps you can explore and 
-modify as you learn about Realm. These apps are a good choice for developers
-who prefer to experiment directly with working code instead of reading a 
-guide that walks you through implementing the code.
+Before You Start
+----------------
 
-Check out the :ref:`Realm template apps <template-apps>`.
+These tutorials start with the Flexible Sync Template App for your chosen language 
```

Comment:
Will this be true for all tuts? I'm not sure we scoped the CLI tutorial as part of this phase 1 project. I could be wrong, though!

Reply:
Everything but Web supports flex sync now, and we have template apps for each of them. So I think this is legit to say. 
Created ticket to make changes to the other repo's page on the Template Apps.